### PR TITLE
feat(.starters): import alpine-starter into `.starters/default`

### PR DESCRIPTION
**Pull request by Tahul**
Resolves https://github.com/nuxtlabs/studio-app/issues/821 for Alpine

You can boot the default starter using `npm run play:starter`.

### CODE REVIEW

File: .starters/default/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Create one with no scope selected on https://github.com/settings/tokens/new
+# This token is used for fetching the repository releases.
+GITHUB_TOKEN=
```
Comment (Atinux) [inline .starters/default/.env.example:3]: Is it necessary? 🤔

Comment (Tahul) [reply .starters/default/.env.example:3]: It was present in the previous Alpine starter, but I guess not.

Removing it!

File: .starters/default/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "nuxi dev",
+    "build": "nuxi build",
+    "generate": "nuxi generate",
+    "preview": "nuxi preview"
+  },
+  "devDependencies": {
+    "nuxt": "3.3.3",
```
[the remark this answers is not in the document]
Comment (Tahul) [reply .starters/default/package.json:12]: It currently breaks the deployments or enforce the usage of `inlineSSRStyles: false`. :(

I think we need the next release of Nuxt 3 which contains the fix!

File: package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "prepare": "nuxi prepare && nuxi prepare .docs",
     "build": "nuxt build .docs",
     "dev": "nuxt dev .docs",
```
Comment (Atinux) [inline package.json:21]:
```suggestion
    "dev": "nuxt dev .starters/default",
```

File: package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
...
+    "play:starter": "nuxt dev .starters/default",
     "lint": "eslint .",
     "generate": "nuxt generate .docs",
```
Comment (Atinux) [inline package.json:24]:
```suggestion
    "generate": "nuxt generate .starters/default",
```

File: package.json [hunk 5]
```diff
@@ -19,6 +19,7 @@
...
     "preview": "nuxt preview .docs",
```
Comment (Atinux) [inline package.json:25]:
```suggestion
    "preview": "nuxt preview .starters/default",
```

File: .starters/default/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Atinux) [reply .starters/default/package.json:12]: I added `inlineSSRStyles: false` in latest Alpine release so it's fine to use latest Nuxt at the moment.

Once the next release of Nuxt is ready we could update Alpine to remove the inlineSSRStyles

**Comment by Tahul** [comment]
@Atinux ; I think for Alpine the `.docs` and `.starters/default` directory are the same.

The only issue that remains is that if we drop `.docs` in favor of `.starters/default`, we won't be able to properly run the theme with the local version of Alpine without the need to overwrite some code everytime we want to boot with such setup.

What should I do for this?

I think I can drop `.docs` and add guidelines for contributing in the readme, or maybe you know a solution?

File: .starters/default/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tahul) [reply .starters/default/package.json:12]: Great!

[the remark this answers is not in the document]
Comment (Tahul) [reply .starters/default/package.json]: Resolved in [772f21e](https://github.com/nuxt-themes/alpine/pull/135/commits/772f21e0e9a74be5c04a046ebd2585991679e5ef)

**Comment by Tahul** [comment]
Maybe I could also work a bit on backend side that makes the copy of our starters and support overwriting the package.json at that time. 🤷

File: package.json (hunk 5 above)
Comment (Tahul) [reply package.json]: Resolved in [9f22735](https://github.com/nuxt-themes/alpine/pull/135/commits/9f2273589705bb534e75a4789f57deb38ba10673)

File: package.json [hunk 6]
```diff
@@ -19,6 +19,7 @@
     "prepare": "nuxi prepare && nuxi prepare .docs",
     "build": "nuxt build .docs",
     "dev": "nuxt dev .docs",
+    "play:starter": "nuxt dev .starters/default",
```
[the remark this answers is not in the document]
Comment (Tahul) [reply package.json]: Resolved in [c0085b1](https://github.com/nuxt-themes/alpine/pull/135/commits/c0085b169a82ea6676b1c74baca431bb44c54d68)

File: package.json (hunk 3 above)
Comment (Tahul) [reply package.json]: Resolved in [`9f22735` (#135)](https://github.com/nuxt-themes/alpine/pull/135/commits/9f2273589705bb534e75a4789f57deb38ba10673)

**Comment by Tahul** [comment]
(The scripts rewrite obviously breaks the deployment, I guess doing deployments from Studio itself is the point of these PRs anyways)

**Comment by larbish** [comment]
What is the issue with the deployment ? What is the difference with https://github.com/Atinux/content-wind (it used the .demo folder in production already) ?

**Comment by Atinux** [comment]
> @Atinux ; I think for Alpine the `.docs` and `.starters/default` directory are the same.
> 
> 
> 
> The only issue that remains is that if we drop `.docs` in favor of `.starters/default`, we won't be able to properly run the theme with the local version of Alpine without the need to overwrite some code everytime we want to boot with such setup.
> 
> 
> 
> What should I do for this?
> 
> 
> 
> I think I can drop `.docs` and add guidelines for contributing in the readme, or maybe you know a solution?

Actually it works well with content-wind thanks to pnpm workspace linking automatically in development so we should not have to change anything normally

**Comment by Atinux** [comment]
So .starters/default is the new .docs

**Pull request merged**